go/vt/vtgate/executorcontext/vcursor_impl.go

```diff
@@ -79,7 +80,7 @@ type (
 
 		MaxMemoryRows      int
 		EnableShardRouting bool
-		DefaultTabletType  topodatapb.TabletType
+		TabletType         dynamicconfig.DefaultTabletType
```

Comment:
vcursor does not need to take a dynamic config. It is per query so we do not want the changed value in between a plan or execution.